sdk/eventhub/Azure.Messaging.EventHubs.Shared/src/Testing/EventHubScope.cs

```diff
@@ -94,7 +108,28 @@ public async ValueTask DisposeAsync()
         }
 
         /// <summary>
-        ///   Performs the tasks needed to create a new Event Hub instance with the requested
+        ///   It returns an EventHubScope after instanciating a management client
+        ///   and querying the consumer groups from the portal.
+        /// </summary>
+        ///
+        /// <returns>The <see cref="EventHubScope" /> that will be used in a given test run.</returns>
+        ///
+        private static async Task<EventHubScope> PopulateAsync()
```

Comment:
nit:  Would you please be so kind as to shift this down?    We're trying to keep members organized by type (fields, properties, constructors, methods, nested types) and by visibility in decreasing order (public, internal, protected, private)